tests/acceptance/cli/test_federated_cli_lifecycle.py

```diff
@@ -1,32 +0,0 @@
-"""
- This file is part of nucypher.
-
- nucypher is free software: you can redistribute it and/or modify
- it under the terms of the GNU Affero General Public License as published by
- the Free Software Foundation, either version 3 of the License, or
- (at your option) any later version.
-
- nucypher is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- GNU Affero General Public License for more details.
-
- You should have received a copy of the GNU Affero General Public License
- along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
-"""
-import pytest_twisted as pt
-
-from tests.acceptance.cli.lifecycle import run_entire_cli_lifecycle
-
-
-@pt.inlineCallbacks
-def test_federated_cli_lifecycle(click_runner,
-                                 random_policy_label,
-                                 federated_ursulas,
-                                 custom_filepath,
-                                 custom_filepath_2):
-    yield run_entire_cli_lifecycle(click_runner,
```

Comment:
Should the `tests/acceptance/cli/lifecycle.py` file also be deleted? I believe the decentralized/federated cli_lifecycle files are the only ones using it.

Reply:
You're right, it can be removed.  I will make an RFC commit.